src/lib/style-util.html

```diff
@@ -142,7 +142,12 @@
           e = e$[i];
           // look inside templates for elements
           if (e.localName === 'template') {
-            cssText += this.cssFromElement(e);
+            // retain css content when specified,
+            // unless the template is subordinate to a <dom-module>
+            if (!e.hasAttribute('preserve-content') ||
+                !element.localName === 'dom-module') {
```

Comment:
Seems like we should remove the check for 'dom-module'. Putting `preserve-content` on a top level template in a `dom-module` is odd, but it seems like we should respect the user's ask here. Perhaps there's an alternate way to supply a template for some reason.